modules/application/src/main/java/edu/northeastern/truthtree/adapter/stories/StoriesDBAdapter.java

```diff
@@ -1,18 +1,80 @@
 package edu.northeastern.truthtree.adapter.stories;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.util.UriComponentsBuilder;
+
 import java.util.List;
 
+import edu.northeastern.truthtree.adapter.utilities.URLUtil;
+import edu.northeastern.truthtree.assembler.StoriesAssembler;
 import edu.northeastern.truthtree.dto.StoryDTO;
+import edu.northeastern.truthtree.enums.OrderType;
+
+import static edu.northeastern.truthtree.AppConst.STORIES_URL_GET;
+import static edu.northeastern.truthtree.AppConst.STORIES_URL_POST;
 
-public class StoriesDBAdapter implements IStoriesAdapter{
+@Component("storiesDBAdapter")
+public class StoriesDBAdapter implements IStoriesAdapter {
+
+  private StoriesAssembler assembler;
+
+  @Autowired
+  public StoriesDBAdapter(StoriesAssembler assembler) {
+    this.assembler = assembler;
+  }
 
   @Override
   public StoryDTO createStory(StoryDTO storyDTO) {
-    return null;
+    ObjectMapper mapper = new ObjectMapper();
+    String jsonString = "";
+    try {
+      jsonString = mapper.writeValueAsString(storyDTO);
+    } catch (JsonProcessingException e) {
+      e.printStackTrace();
```

Comment:
Let's throw the exception here so that we can return a bad request error at controller level.

Reply:
We still have to add exception framework.